pkgpanda/build/__init__.py

```diff
@@ -773,6 +774,10 @@ def cache_abs(filename):
 
     final_buildinfo['sources'] = sources
 
+    if builder.has('sysctl'):
+        sysctl = builder.take('sysctl')
+        final_buildinfo['sysctl'] = sysctl
```

Comment:
You  shouldn't touch final_buildinfo here. The ones that do are ones which modify the value given. With things like `sources` we perform expansion / resolution to a certain level (Checkout the git repo, etc, expand `single_source` to be `sources`, etc.). This one we don't, so it doesn't need to be there.